patches/sitecustomize.py

```diff
@@ -101,10 +101,11 @@ def new_configure(*args, **kwargs):
             client_options = kwargs['client_options']
         else:
             client_options = {}
-        client_options['api_endpoint'] = os.environ['KAGGLE_DATA_PROXY_URL']
+        client_options['api_endpoint'] = os.environ['KAGGLE_GRPC_DATA_PROXY_URL']
         if os.getenv('KAGGLE_GOOGLE_GENERATIVE_AI_USE_REST_ONLY') != None:
             client_options['api_endpoint'] += '/palmapi'
             kwargs['transport'] = 'rest'
+            client_options['api_endpoint'] = os.environ['KAGGLE_DATA_PROXY_URL']
```

Comment:
We would want this after line 111 as well, because that is also specifying a REST transport

Reply:
@Philmod I applied the changes from this comment into the branch

Reply:
Thanks, I simplified a bit the code.